<link href="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.min.css" rel="stylesheet" id="bootstrap-css">
<script src="//netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/js/bootstrap.min.js"></script>
<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
<!------ Include the above in your HEAD tag ---------->
<style>
section {
  background-color: rgb(255, 255, 255);
  padding: 10px;
  margin-bottom: 10px;
}
h1,h2,h3,h4,h5,h6{
    padding : 5px;
    margin-bottom: 5px;
    
}
img{
    margin-top : 15px;
    margin-bottom: 15px;
}
p{
    padding : 5px;
    margin-top: 5px;
    margin-bottom: 5px;
    font-size: 15px;
    font-weight: bold;
    color: rgb(100, 100, 100);
}
hr{
    margin-bottom: 5px;
}
</style>
<div class="container" style ="width:90%; height:100%;">
    <div class ="manual-content" style="
        position: absolute;
        top: 10px;left: 400px;">
        <div id="contents-header">
        <h1>About Group Comparison</h1><hr>
        <h4>
            The group comparison function allows you to compare your desired target with meta information from the NMF cluster (ex: sex, DX, Subtype, etc.) through grouping.
        </h4>

        <h5>The target can be chosen from the following categories: Protein, RNA, Phosphosite, or Acetylsite.</h5>
        <p>The t-test (Test statistic) is divided by the number of groups.</p>
        <p>If the number of groups is 2, the Wilcoxon Rank-Sum Test (= Mann-Whitney U-test) is used.</p>
        <p>Otherwise, if the number of groups is over 3, the Kruskal-Wallis test (= One way ANOVA on rank) is used.</p>
        <p>The p-value represents the difference between each meta data of values.</p>
        <p>In the meta columns, for the 'Subtype' and 'Histology' plots, normal data is included, and a boxplot is drawn. </p>
        <p>In the 'Subtype', a value of "0" indicates normal data, while in the 'Histology', a value of "Normal" represents normal data.</p>
        <p>In the 'Subtype Core', only data with a Subtype Membership Score above 0.5, indicated by a Subtype Core value of 'Y,' are selected. 
            A boxplot is then generated for each Subtype.</p>
        </p>

        </div>
        <div class="section1">
            <h3>Basic functions</h3> <hr>
            <section>
                <h4>1. Using Group Comparison</h4> <hr>
                <p>In the first box (1) as shown below, select your target. The second box (2) is then used to choose the meta information for it to be compared to.</p> 
                <p>Press the [Apply] button to see results.</p>
                <img style="width:1068px" alt="group_comparison_apply" src="../file/assets/img/group_comparison/group_comparison1.png">
                <hr>
            </section>

            <section>
                <h4>2. Number of data used</h4> <hr>
                <p>You can check the number of data used in the boxplot.</p>
                <img style="width:1068px" alt="group_comparison_apply" src="../file/assets/img/group_comparison/group_comparison_counting.png">
                <hr>
            </section>

            <section>
                <h4>3. Result Sorting</h4><hr>
                <p>Search results can also be organized by its P-Value in the same type data.</p>
                <img style="width:1068px" alt="group_comparison_sort" src="../file/assets/img/group_comparison/group_comparison_sort.png">
                <hr>
            </section>

            <section>
                <h4>4. Legend information</h4>
                <hr>
                <table class="table table-bordered">
                    <tbody>
                        <tr>
                            <th>Subtype</th>
                            <td>Korean NSCLC Subtype (0 to 5, 0 is normal).</td>
                        </tr>
                        <tr>
                            <th>Subtype Core</th>
                            <td>Korean NSCLC Subtype (1 to 5) with only subtype core sample. 
                                The subtype core is determined based on each sample’s subtype membership score. 
                                Samples with a membership score of 0.5 or higher are classified as the core of the subtype and labeled “Y”, while those with a score below 0.5 are labeled “N”.</td>
                        </tr>
                        <tr>
                            <th>Histology</th>
                            <td>Name of pathological diagnosis classification for using this study.<p></p>
                                (AD :Adenocarcinoma , SC : Squamous cell carcinoma, MA : Mucinous adenocarcinoma, NC : Large cell neuroendocrine carcinoma  and others, Normal: normal)</td>
                        </tr>
                        <tr>
                            <th>Sex</th>
                            <td>Sex (F: Female, M: Male).</td>
                        </tr>
                        <tr>
                            <th>Smoking</th>
                            <td>Status of smoking history on samples.
                            <p></p>(Cr: current smoker; Ex: quit smoking more than a year before sugery; N: non-smoker; NA: no information)</td>
                        </tr>
                        <tr>
                            <th>TNM Stage</th>
                            <td>The 8th Edition of the American Joint Committee on Cancer (AJCC-TNM) stage.</td>
                        </tr>
                        <tr>
                            <th>Pathologic N</th>
                            <td>Status of node metastasis on samples.<p></p>
                                (N0: no lymph node metastasis; N1: eastern pulmonary lymph node metastasis ; N2: eastern mediastinal lymph node metastatis)</td>
                        </tr>
                        <tr>
                            <th>Adjuvant Treatment</th>
                            <td>Status of adjuvant treatment on samples.<p></p>
                                (CTx: Chemotherapy; RTx: Radiation therapy; None: no treatment)</td>
                        </tr>
                        <tr>
                            <th>Recurrence Status</th>
                            <td>Status of recurrence on samples.<p></p>
                                 (1: Recur occur; 0: Not recur occur; NA: No data)</td>
                        </tr>
                        <tr>
                            <th>TIL Pattern</th>
                            <td>The pattern of tumor infiltrating lymphocytes (TIL).</td>
                        </tr>
                        <tr>
                            <th>Immune Cluster</th>
                            <td>Cell type-based Immune Cluster (Hot, Cold).</td>
                        </tr>
                        <tr>
                            <th>Whole Genome Doubling</th>
                            <td>Presence of whole genome doubling.</td>
                        </tr>
                        <tr>
                            <th>TP53</th>
                            <td>TP53 mutation status which contains CNV loss and truncating mutation or both.</td>
                        </tr>
                        <tr>
                            <th>Other Tumor Suppressor genes</th>
                            <td>Other tumor suppressor genes mutation status which contains CNV loss and truncating mutation or both. <p></p>
                                Other tumor suppressor genes consisted of CDKN2A, STK11, KEAP1, RB1, PPP2R1A, and SMARCA4.</td>
                        </tr>
                        <tr>
                            <th>EGFR</th>
                            <td>EGFR mutation status which contains L858R, exon19 deletion, other indels and other SNVs.</td>
                        </tr>
                        <tr>
                            <th>Other Oncogene Alteration</th>
                            <td>Other oncogenes mutation or fusion which contains KRAS, PIK3CA ERBB2, MET, ALK, ROS1, RET.</td>
                        </tr>
                    </tbody>
                </table>
                <hr>
            </section>

        </div>
        <div class="section2">
            <h3>Tab funtions</h3><br>
            <section>
                <h4>1. Image Size Adjustment</h4><hr>
                <p>How large the image appears on the screen can be adjusted using the zoom in and out buttons.</p>
                <img style="width:1068px" alt="group_comparison_zoom" src="../file/assets/img/group_comparison/group_comparison_zoom.png">
            </section>

            <section>
                <h4>2. Saving Visualized Data</h4><hr>
                <p>The resulting plot can be saved by pressing the save image button as shown below.</p>
                <img style="width:1068px" alt="group_comparison_save_img" src="../file/assets/img/group_comparison/group_comparison_saveimg.png">               
                <hr>
            </section>
        </div>
    </div>
	<div class="row">
        <div style="width:300px; padding: 8px 0; left:3px; position:fixed;">
            <div style="overflow-y: scroll; overflow-x: hidden; height: 100%;">
                <ul class="nav nav-list">
                    <li><label class="tree-toggler nav-header">NMF Cluster</label>
                        <ul class="nav nav-list tree">
                            <li><a href="/subpage/nmf_help.html">NMF cluster</a></li>
                        </ul>
                    </li>

                    <li class="divider"></li>
                    <li><label class="tree-toggler nav-header">Immune Cluster</label>
                        <ul class="nav nav-list tree">
                            <li><a href="/subpage/immune_help.html">Immune Cluster</a></li>
                        </ul>
                    </li>

                    <li class="divider"></li>
                    <li><label class="tree-toggler nav-header">Quantitation</label>
                        <ul class="nav nav-list tree">
                            <li><a href="/subpage/quantification_protein_help.html">Protein</a></li>
                        </ul>
                        <ul class="nav nav-list tree">
                            <li><a href="/subpage/quantification_phosphosite_help.html">Phosphosite</a></li>
                        </ul>
                        <ul class="nav nav-list tree">
                            <li><a href="/subpage/quantification_acetylsite_help.html">Acetylsite</a></li>
                        </ul>
                        <ul class="nav nav-list tree">
                            <li><a href="/subpage/quantification_rna_help.html">RNA</a></li>
                        </ul>
                    </li>

                    <li class="divider"></li>
                    <li><label class="tree-toggler nav-header">Group Comparison</label>
                        <ul class="nav nav-list tree">
                            <li class = item-1 current active>
                                <a class="" href="/subpage/group_comparison_help.html" style="background-color:antiquewhite;">
                                    <span class="sign"><i class="icon-play"></i></span>
                                    <span class="lbl">Group Comparison</span> (current)
                                </a>
                            </li>
                        </ul>
                    </li>

                    <li class="divider"></li>
                    <li><label class="tree-toggler nav-header">Survival Analysis</label>
                        <ul class="nav nav-list tree">
                            <li><a href="/subpage/survival_help.html">Survival Analysis</a></li>
                        </ul>
                    </li>
                    <li class="divider"></li>
                </ul>
            </div>
        </div>
	</div>
</div>
<script>
    $(document).ready(function () {
    $('label.tree-toggler').click(function () {
        $(this).parent().children('ul.tree').toggle(300);
    });
});
</script>
